docs/ignoring-files-folders-code.md

```diff
@@ -96,9 +96,13 @@ specification more closely.
 
 ## Define ignored files and folders in Semgrep AppSec Platform
 
-Another method for users to define ignore patterns is through a Project in Semgrep AppSec Platform. These patterns follow the same syntax as `.semgrepignore` in the preceding section.
+Another method for users to define ignore patterns is through a Project in Semgrep AppSec Platform. These patterns follow the same syntax as `.semgrepignore` in the preceding section. You can define patterns for individual projects, or you can define them at the organization level so that they're applied to all projects owned by that organization.
 
-To define files and folders in Semgrep AppSec Platform:
+Including files and folders through this method is **additive**. When you run a scan using `semgrep ci`, Semgrep looks for a `.semgrepignore` within the repository. If no `.semgrepignore` file is found, Semgrep temporarily creates one and adds items from Semgrep AppSec Platform's Path Ignores.
+
+Adding items to Semgrep AppSec Platform's **Path Ignores** box doesn't override default Semgrep ignore patterns included with its CLI tool, since the patterns are additive. However, items added to `.semgrepignore` override default Semgrep CLI patterns.
```

Comment:
This has nothing to do with a TW review, but it has always tripped me that "x overwrites y but z does not overwrite y". I think the best practice here is to stick to 1 source for defining ignores but I also know that historically there are edges where defining in CLI is more granular somehow than in the Platform. (No action needed, but ... idk, every time I touch/see the ignore documentation, it crosses my mind.)